de.tudarmstadt.ukp.wikipedia.api/pom.xml

```diff
@@ -25,6 +25,16 @@
 		<artifactId>de.tudarmstadt.ukp.wikipedia</artifactId>
 		<version>1.2.0-SNAPSHOT</version>
 	</parent>
+
+    <properties>
+        <!--
+            By default, we skip the integration tests for regular contributors
+            as they might not have access to one of the two institution specific
+            MySQL resource profiles.
+        -->
+        <skip.integration.tests>true</skip.integration.tests>
```

Comment:
I suppose you could just set the `skipITs` property directly. Cf. https://maven.apache.org/surefire/maven-failsafe-plugin/integration-test-mojo.html#skipITs

Reply:
@reckart  9bb1e7e fixed it, so it works fine for devs outside UKP/HHN. People or CI-servers inside UKP/HHN can now choose to run ITs. See my updated comment above.